angular/wolox-books/src/styles.scss

```diff
@@ -1 +1,8 @@
 /* You can add global styles to this file, and also import other style files */
+@import 'app/scss/equalizer';
+@import 'app/scss/fonts';
```

Comment:
you should create an `application.scss` that only has imports (change the config in the angular.json to use that file instead of this one)